src/averageAges.js

```diff
@@ -20,6 +20,13 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let men;
+
+  (arguments.length < 2) ? (men = people.filter(person => person.sex === 'm'))
```

Comment:
check if you have `century` not the length of arguments and formate your code to make it more readable